tests/unit/services/group.test.js

```diff
@@ -0,0 +1,187 @@
+import Group from '../../../src/db/models/group';
+import * as GroupService from '../../../src/services/group';
+
+
+describe('Test the groups service', () => {
+    test('It should return all groups', async () => {
+        // Mock Model findAll
+        jest.spyOn(Group, 'findAll').mockImplementation(() => {
+            return Promise.resolve([
+                {
+                    'id': '79836ba5-e47f-4322-a498-53e7744189ad',
+                    'name': 'Read only',
+                    'permissions': [
+                        'READ'
+                    ]
+                },
+                {
+                    'id': 'f5637788-3889-4a6e-9f5f-f38a1a144124',
+                    'name': 'Read/Write',
+                    'permissions': [
+                        'READ',
+                        'WRITE'
+                    ]
+                }
+            ]);
+        });
+
+        const output = await GroupService.findAll();
+
+        expect(output).toEqual(expect.arrayContaining(
+            [
+                expect.objectContaining({
+                    'id': '79836ba5-e47f-4322-a498-53e7744189ad',
+                    'name': 'Read only',
+                    'permissions': [
+                        'READ'
+                    ]
+                })
+            ])
+        );
+    });
+
+    test('It should return individual group', async () => {
+        // Mock Model findOne
+        jest.spyOn(Group, 'findOne').mockImplementation(() => {
+            return Promise.resolve(
+                {
+                    'id': '79836ba5-e47f-4322-a498-53e7744189ad',
+                    'name': 'Read only',
+                    'permissions': [
+                        'READ'
+                    ]
+                }
+            );
+        });
+
+        const output = await GroupService.findById('79836ba5-e47f-4322-a498-53e7744189ad');
```

Comment:
Consider using constants and functions to reduce repeated values and object and reuse as much as possible.